bls/src/main/java/org/dashj/bls/DASHJBLS.java

```diff
@@ -0,0 +1,104 @@
+/* ----------------------------------------------------------------------------
+ * This file was automatically generated by SWIG (http://www.swig.org).
+ * Version 4.0.2
+ *
+ * Do not make changes to this file unless you know what you are doing--modify
+ * the SWIG interface file instead.
+ * ----------------------------------------------------------------------------- */
+
+package org.dashj.bls;
+
+public class DASHJBLS {
+  public static boolean objectEquals(ChainCode a, ChainCode b) {
+    return DASHJBLSJNI.objectEquals__SWIG_0(ChainCode.getCPtr(a), a, ChainCode.getCPtr(b), b);
+  }
```

Comment:
This class contains many methods that didn't end up in a Java class.

`objectEquals` is wrapping `operator==` from the C++ classes.  For some reason, the SWIG tool doesn't convert these into `equals`